easybuild/easyconfigs/f/FLASH/FLASH-2.2.00-foss-2022b.eb

```
@@ -0,0 +1,34 @@
+easyblock = 'MakeCp'
+
+name = 'FLASH'
+version = '2.2.00'
+
+homepage = 'https://ccb.jhu.edu/software/FLASH/'
+description = """FLASH (Fast Length Adjustment of SHort reads) is a very fast
+and accurate software tool to merge paired-end reads from next-generation
+sequencing experiments. FLASH is designed to merge pairs of reads when the
+original DNA fragments are shorter than twice the length of reads. The
+resulting longer reads can significantly improve genome assemblies. They can
+also improve transcriptome assembly when FLASH is used to merge RNA-seq data.
+"""
+
+toolchain = {'name': 'foss', 'version': '2022b'}
+
+source_urls = ['https://github.com/dstreett/FLASH2/archive/']
+sources = ['%(version)s.zip']
+checksums = ['1e54b2dd7d21ca3e0595a3ffdd27ef3098f88c4de5b9302ec5ea074b49b79960']
+
```

(explicit `zlib` and `binutils` dependencies needed when dropping to `GCCcore` )
```suggestion

builddependencies = [('binutils', '2.39')]

dependencies = [('zlib', '1.2.12')]

```